src/tutorial/scheduling/further-scheduling.rst

```diff
@@ -53,38 +53,6 @@ to handle events within your code (custom outputs).
    *For more information see the* `Cylc User Guide`_.
 
 
-.. _tutorial-clock-triggers:
-
-Clock Triggers
---------------
```

Comment:
There is still a more detailed clock trigger tutorial in Further Topics. Perhaps keep this section but just say something like:

> We have already introduced clock triggers in the :ref:`tutorial-datetime-cycling` tutorial. See the :ref:`tutorial-cylc-clock-trigger` tutorial for more advanced use cases.